### PR TITLE
[DOCFIX] Remove ALLUXIO_RELEASED_VERSION from docs/_config.yml

### PR DESCRIPTION
**Pull request by Xenorith**
Update references to ALLUXIO_RELEASED_VERSION to be ALLUXIO_VERSION_STRING
Remove the even more obscure ALLUXIO_MASTER_VERSION_SHORT
Update ALLUXIO_MAJOR_VERSION on master to be "edge"

The concept of "most recently release version" changes as patches are released, but this update is not reflected outside of the branch where the patch was cut from
Instead of trying to update ALLUXIO_RELEASED_VERSION across multiple branches, use ALLUXIO_VERSION_STRING which will be correct in all branches except for master.
In the case of master, which is currently at 2.2.0-SNAPSHOT, it is likely there will be invalid references (ex. <version>2.2.0-SNAPSHOT</version> in the maven dependency declaration).
But I find this is to be more acceptable since it is a development branch.

In summary, this PR introduces inconsistencies in the master branch but corrects and simplifies version strings across all other release branches.

### CODE REVIEW

**Comment by Xenorith** [comment]
@apc999 would like your input and approval from a community perspective
@ZacBlanco hope this helps out with the version script
@LuQQiu can you look over my chinese edits? do not trust google translate

please do not auto cherry pick this to other branches. after approvals, i will manually cherry pick this over to branch-2.1 and branch-2.0 because not all changes here will apply to release branches

File: docs/cn/contributor/Building-Alluxio-From-Source.md [hunk 1]
```diff
@@ -23,10 +23,11 @@ priority: 0
 $ git clone git://github.com/alluxio/alluxio.git
 $ cd alluxio
 ```
-您可以编译特定版本的Alluxio，例如{{site.ALLUXIO_RELEASED_VERSION}}。否则这将编译源码的master分支。
+您可以编译特定版本的Alluxio否则这将编译源码的master分支。
```
Comment (LuQQiu) [inline docs/cn/contributor/Building-Alluxio-From-Source.md]: 您可以编译特定版本的Alluxio否则这将编译源码的master分支。 -> 您可以编译特定版本的Alluxio`，`否则这将编译源码的master分支。

File: docs/cn/overview/Getting-Started.md [hunk 2]
```diff
@@ -35,16 +35,16 @@ priority: 1
 
 ## 下载 Alluxio
 
-首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认 Hadoop 版本构建的{{site.ALLUXIO_RELEASED_VERSION}}预编译版。
+首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认Hadoop版本构建的相应预编译版。
```
Comment (LuQQiu) [inline docs/cn/overview/Getting-Started.md]: `首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。选择为默认Hadoop版本构建的相应预编译版。`
-> `首先，从[这里](http://www.alluxio.io/download)下载 Alluxio。`
The following line can be removed since we only have the default hadoop version in download page

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please
